exp/api/v1beta1/finalizers.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+const (
+	// FargateProfileFinalizer allows the controller to clean up resources on delete.
+	FargateProfileFinalizer = "awsfargateprofile.infrastructure.cluster.x-k8s.io"
+
+	// MachinePoolFinalizer is the finalizer for the machine pool.
+	MachinePoolFinalizer = "awsmachinepool.infrastructure.cluster.x-k8s.io"
+
+	// ManagedMachinePoolFinalizer allows the controller to clean up resources on delete.
+	ManagedMachinePoolFinalizer = "awsmanagedmachinepools.infrastructure.cluster.x-k8s.io"
```

Comment:
Just for reference: These constants existed before this PR. In this PR, they are moved into one file.

Reply:
Thats correct, they were moved as we added a new finalizer....which i have now removed. But it feels like a refactor to keep.